modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/cards/Cards.js

```diff
@@ -29,7 +30,11 @@ const Cards = ({items, schema}) => {
 							className="col-md-3"
 							key={item[selectedItemsKey] || index}
 						>
-							<Card item={item} schema={schema} />
+							<Card
```

Comment:
Just a quick fix to display item actions icons in the card component. This is something that we could provide up in the context and maybe sometime make it customizable.

Reply:
Feel free to go nuts with `cards` view implementation. I don't think it's used anywhere in DXP, apart from the sample module. I suspect there are a lot of small issues like this, missing API implementations etc.